price_service/server/src/rest.ts

```diff
@@ -523,31 +523,17 @@ export class RestAPI {
     endpoints.push("ready");
 
     app.get("/live", (_, res: Response) => {
-      const threshold = 60;
-      const stalePriceTreshold = 10;
-      const minimumNumPrices = 100;
-
-      const currentTime: TimestampInSec = Math.floor(Date.now() / 1000);
-
-      const priceIds = [...this.priceFeedVaaInfo.getPriceIds()];
-      let stalePriceCnt = 0;
-
-      for (const priceId of priceIds) {
-        const latency =
-          currentTime -
-          this.priceFeedVaaInfo.getLatestPriceInfo(priceId)!.attestationTime;
-        if (latency > threshold) {
-          stalePriceCnt++;
-        }
-      }
-
-      if (
-        priceIds.length < minimumNumPrices ||
-        stalePriceCnt > stalePriceTreshold
-      ) {
-        res.sendStatus(StatusCodes.SERVICE_UNAVAILABLE);
-      } else {
+      // Reuse the same readiness check for liveness.
```

Comment:
Just an observation, but I think there are some assumptions about how the price service is deployed in this comment.

Reply:
Right. I reverted the liveness back to be always Ok. instead in our infra we will use readiness check for liveness. This seems better.